polygeist/tools/cgeist/Test/Verification/sycl/functions.cpp

```diff
@@ -40,6 +40,20 @@
 
 template <typename T> SYCL_EXTERNAL void keep(T);
 
+// When DPC++ RT accessor::get_pointer returns a pointer, which matches the interface of sycl.accessor.get_pointer:
```

Comment:
Suggestion:

// COM: Commenting out the checks below, this is the code that should be generated.  Currently the DPC++ SYCL RT implementation of accessor::get_pointer is non-conforming. Once that problem is fixed we enable the checks  below.